docs/i18n/fr/docusaurus-plugin-content-docs/current/usage/getting-started.mdx

```diff
@@ -1,4 +1,4 @@
-Voici la traduction en français, en préservant le formatage Markdown, les blocs de code et le front matter, ainsi que les balises {% jsx %} et similaires. Les exemples de code, les URLs et les termes techniques n'ont pas été traduits.
+
```

Comment:
😂

Reply:
This is where tool use might come in handy! It shouldn't respond like it responds to a user 😂